Helpers/Eligibility.php

```diff
@@ -309,6 +319,6 @@ public function isEligible()
      */
     public function getMessage()
     {
-        return $this->message;
+        return __($this->message);
```

Comment:
Maybe add title in doc block about to inform this method `Get a translated message`